packages/rabbitmq/test/rabbitmqTxSubmitProvider.test.ts

```diff
@@ -0,0 +1,66 @@
+import { TxSubmitProvider } from '@cardano-sdk/core';
+import { rabbitmqTxSubmitProvider } from '../src';
+
+const BAD_CONNECTION_URL = new URL('amqp://localhost:1234');
+const GOOD_CONNECTION_URL = new URL('amqp://localhost');
+
+describe('rabbitmqTxSubmitProvider', () => {
+  let provider: TxSubmitProvider;
+
+  afterEach(() => provider?.close!());
+
+  describe('healthCheck', () => {
+    it('is not ok if cannot connect', async () => {
+      provider = rabbitmqTxSubmitProvider(BAD_CONNECTION_URL);
+      const res = await provider.healthCheck();
+      expect(res).toEqual({ ok: false });
+    });
+
+    it('is ok if cannot connect', async () => {
+      provider = rabbitmqTxSubmitProvider(GOOD_CONNECTION_URL);
+      // Called twice to cover AMQPWrapper.#connect() first line as well
+      const resA = await provider.healthCheck();
+      const resB = await provider.healthCheck();
+      expect(resA).toEqual({ ok: true });
+      expect(resB).toEqual({ ok: true });
+    });
+  });
+
+  describe('submitTx', () => {
+    it('resolves if successful', async () => {
+      const success = jest.fn();
```

Comment:
A cleaner approach here is to declare the expected number of assertions instead of these spys:

``` TypeScript
expect.assertions(2)
```

Reply:
I changed this but I'm not going to resolve in order to check my changes in next commits.

Reply:
We clarified this in next suggestions